extensions/tmtv/src/utils/hpViewports.ts

```diff
@@ -0,0 +1,438 @@
+const ctAXIAL = {
```

Comment:
This is a nice way of defining things so that the hanging protocol doesn't get quite so big.